automatic/wps-office-free/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,22 @@
+
+  $version = '10.2.0.5820'
+  $regName = $packageName -replace('\-',' ')
+  $regName = $regName -replace('free','*')
+  $registry = Get-UninstallRegistryKey -SoftwareName $regName
+  
+  $packageArgs = @{
+    packageName   = 'wps-office-free'
+    fileType      = 'exe'
+    url           = 'https://jump.wps.com/latest_package?distsrc=00100.00000103&lang=en'
+    silentArgs    = '/S'
+    validExitCodes= @(0)
+    softwareName  = 'WPS Office 2016 Free'
+    checksum      = '1F2F28BE2A2E21D2CF0B3FA0E3196D2291470220A28645CADBCC55EECD936C3B'
+    checksumType  = 'sha256'
+  }
+  
+  if ( $version -ge $registry.DisplayVersion) {
```

Comment:
Is the package that large that it really needs to be checked if it is already installed?
I.E above 100 MB